.github/workflows/_build-python-wheel-extension.yml

```diff
@@ -173,9 +171,9 @@ jobs:
           - macos-15-intel
           - windows-latest
         python:
+          - "3.11"
```

Comment:
The test should be on the same matrix as the build, so we need the same code as above here